### DISCUSSION TITLE
NSwag fails at compile time when using Docker to run the database

### DISCUSSION THREAD

**Discussion by apolonas** [jasontaylordev/CleanArchitecture#1161] [General]
Everything was working fine until I Doeckerized the API app to run inside a container, including the underlying database. NSwag fails to connect to the database when I rebuild the code and throws an error that prevents the app from building. Is there a workaround for this?

**Comment by jasontaylordev** [comment]
Try setting the environment variable `SkipNSwag = False`, this will result in NSwag not running on build. See `src/Web/Web.csproj` for more information.
